src/prebid.js

```diff
@@ -1090,4 +1090,19 @@ $$PREBID_GLOBAL$$.processQueue = function () {
   processQueue($$PREBID_GLOBAL$$.cmd);
 };
 
+/**
+ * @alias module:pbjs.triggerBilling
+ */
+$$PREBID_GLOBAL$$.triggerBilling = (bidder, bid, adUnits, am = adapterManager) => {
```

Comment:
this is a public method, so its interface should be kept clean - I don't think you need anything more than `bid`.